<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en">
	<head>
		
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
		<meta name="robots" content="index, follow" />
		<meta name="generator" content="RapidWeaver" />
		
	<meta name="twitter:card" content="summary">
	<meta name="twitter:title" content="Research | Schmitz Lab Website">
	<meta name="twitter:url" content="https://schmitzlab.uga.edu/research/research.html">
	<meta property="og:type" content="website">
	<meta property="og:site_name" content="Schmitz Lab Website">
	<meta property="og:title" content="Research | Schmitz Lab Website">
	<meta property="og:url" content="https://schmitzlab.uga.edu/research/research.html">
		<title>Research | Schmitz Lab Website</title>
		<link rel="stylesheet" type="text/css" media="all" href="../rw_common/themes/cameo/consolidated-1.css?rwcache=735837025" />
		
		<!--[if lte IE 7]><link rel="stylesheet" type="text/css" media="screen" href="../rw_common/themes/cameo/css/ie/ie.css" />
		<![endif]-->
		<!--[if IE 6]><link rel="stylesheet" type="text/css" media="screen" href="../rw_common/themes/cameo/css/ie/ie6.css" />
		<![endif]-->
		
		
		
		
		
		
		
		<script type="text/javascript" src="../rw_common/themes/cameo/javascript.js?rwcache=735837025"></script>
		
		
<!--[if IE 6]><script type="text/javascript" charset="utf-8">
			var blankSrc = "../rw_common/themes/cameo/png/blank.gif";
		</script>	
		<style type="text/css">

		img {
			behavior:	url("../rw_common/themes/cameo/png/pngbehavior.htc");
		}

		</style><![endif]-->
		
				<link rel='stylesheet' type='text/css' media='all' href='../rw_common/plugins/stacks/stacks.css?rwcache=735837025' />
		<link rel='stylesheet' type='text/css' media='all' href='files/stacks_page_page1.css?rwcache=735837025' />
        
        
        
        
		
        <meta name="formatter" content="Stacks v4.3.0 (6300)" >
		

		
	</head>
<body>
<div id="container"><!-- Start container -->
	<div id="pageHeader"><!-- Start page header -->
		<div id="grad">
			<img src="../rw_common/themes/cameo/images/header_bottom_grad.png" alt="" style="width: 3000px; height: 66px;" />
		</div>
		
		<h1>Schmitz Lab Website</h1>
		<h2></h2>
	</div><!-- End page header -->
	<div id="navcontainer"><!-- Start Navigation -->
		<div id="top_grad">
			<img src="../rw_common/themes/cameo/images/menu_top_grad.png" alt="" style="width: 3000px; height: 32px;" />
		</div>
		<ul><li><a href="../" rel="">Home</a></li><li><a href="research.html" rel="" id="current">Research</a></li><li><a href="../publications/publications.html" rel="">Publications</a></li><li><a href="../people/people.html" rel="">People</a></li><li><a href="../data/data.html" rel="">Data</a></li><li><a href="../news/news.html" rel="">News</a></li><li><a href="../join/join.html" rel="">Join</a></li><li><a href="../contact/contact.html" rel="">Contact</a></li></ul>
		<div class="clearer"></div>
	</div><!-- End navigation -->
	<div id="sidebarContainer"><!-- Start Sidebar wrapper -->
		<div id="sidebar"><!-- Start sidebar content -->
			<h1 class="sideHeader"></h1><!-- Sidebar header -->
			<!-- sidebar content you enter in the page inspector -->
			 <!-- sidebar content such as the blog archive links -->
		</div><!-- End sidebar content -->
	</div><!-- End sidebar wrapper -->
	<div id="contentContainer"><!-- Start main content wrapper -->
		<div id="content"><!-- Start content -->
			
<div id='stacks_out_1' class='stacks_top'><div id='stacks_in_1' class=''><div id='stacks_out_2' class='stacks_out'><div id='stacks_in_2' class='stacks_in text_stack'>We are interested in determining how phenotypic variation is driven by natural and spontaneous epigenetic variation. Recent advances in genomic technologies are enabling acquisition of sequence-level data at an unprecedented rate and resolution. As a result, genomes for thousands of individuals are being analyzed to determine the total genetic variation present within a species and to determine the impact these variants have on phenotypic variation. Missing from these efforts is the identification of environmentally induced, spontaneous and natural epigenetic alleles (epialleles). <br /><br />We apply epigenomic approaches to populations to study the impact that epialleles have on life history traits and to understand their role in establishing responses to the environment. These epialleles are often associated with changes in their DNA methylation status making their identification possible. Systematically identifying these epialleles, understanding their patterns of heritability, their interaction with genetic variants and their specific roles in controlling gene expression is necessary for a comprehensive understanding of phenotypic variation.<br /><br /><br /></div></div><div id='stacks_out_3' class='stacks_out'><div id='stacks_in_3' class='stacks_in image_stack'>
<div class='centered_image' >
    <img class='imageStyle' src='files/stacks-image-a032e79.jpg' alt='Stacks Image 16' />
</div>

</div></div><div id='stacks_out_4' class='stacks_out'><div id='stacks_in_4' class='stacks_in text_stack'><span id='stacks_in_17'><p style="text-align:center;">Natural variation of DNA methylation in a population of Arabidopsis thaliana. <br />Above is a representative image from a genome browser displaying whole-genome bisulfite sequencing data. <br /><br /><br /></p></span></div></div><div id='stacks_out_5' class='stacks_out'><div id='stacks_in_5' class='stacks_in text_stack'>We are also applying epigenomic profiling methods to discover non-coding regulatory elements across numerous plant species. These methods include genome-wise assays to functionally evaluate identified regions using both gain-of-function and loss-of-function approaches. With advances in genome-editing technologies we are beginning to engineer non-coding regulatory elements as well as to use identified sequences to create modular units that can be used to drive transgene expression in a predictable manner.<br /><br /></div></div><div id='stacks_out_6' class='stacks_out'><div id='stacks_in_6' class='stacks_in image_stack'>
<div class='centered_image' >
    <img class='imageStyle' src='files/stacks-image-1eafad8-1200x630.png' alt='Stacks Image 19' />
</div>

</div></div><div id='stacks_out_7' class='stacks_out'><div id='stacks_in_7' class='stacks_in text_stack'><span id='stacks_in_20'><p style="text-align:center;">Combining nuclei sorting with ATAC-seq reveals regions of chromatin accessibility and candidate cis-regulatory elements in plant genomes. The distance of accessible chromatin and their associated cis-regulatory elements to their nearest genes is greater in plants with larger genomes. <br /><br /></p></span></div></div><div id='stacks_out_8' class='stacks_out'><div id='stacks_in_8' class='stacks_in text_stack'><h3>Epigenetics</h3></div></div><div id='stacks_out_9' class='stacks_out'><div id='stacks_in_9' class='stacks_in text_stack'><br /></div></div><div id='stacks_out_10' class='stacks_out'><div id='stacks_in_10' class='stacks_in float_left_stack'><div class='stacks_div stacks_float_left'>
<div class='centered_image' >
    <img class='imageStyle' src='files/stacks-image-6ccda3e.jpg' alt='Stacks Image 24' />
</div>

</div>The term "epigenetic" was originally defined to convey how an organism of a single genotype can develop from undifferentiated cells: essentially defining a developmental program. This definition requires that gene expression states are heritable through mitotic cell divisions, but there is growing interest in understanding how gene expression states are heritable through meiotic cell divisions. We are using epigenomic approaches to profile diverse plant populations that have been collected over generational time to identify epialleles and to understand mechanisms important for stable transmission of these epiallelic states. Upon identification of epialles from populations over generational time we aim to explore the mechanisms that are essential to maintaining epiallelic states. The above figure demonstrates one possible mechanism that can lead to an epiallelic state. In this scenario, the maintenance methyltransferase MET1 fails to accurately maintain CG methylation which results in a breakdown of information flow between generations.  <br /><br /></div></div><div id='stacks_out_11' class='stacks_out'><div id='stacks_in_11' class='stacks_in text_stack'><br /></div></div><div id='stacks_out_12' class='stacks_out'><div id='stacks_in_12' class='stacks_in text_stack'><h3>Population Epigenomics</h3></div></div><div id='stacks_out_13' class='stacks_out'><div id='stacks_in_13' class='stacks_in text_stack'><br /></div></div><div id='stacks_out_14' class='stacks_out'><div id='stacks_in_14' class='stacks_in float_right_stack'><div class='stacks_div stacks_float_right'>
<div class='centered_image' >
    <img class='imageStyle' src='files/stacks-image-a20f379.jpg' alt='Stacks Image 29' />
</div>

</div>The lab is using a variety of epigenomic approaches in combination with diverse populations of plants to understand the impact of natural DNA methylation variation on phenotypic variation using three major approaches:<br /><br /><strong>1. methylQTL <br /></strong>Genome-wide association studies (GWAS) is a powerful approach for identifying the genetic basis for phenotypic variation within large populations. We are interested in understanding how epialleles for in natural populations and to accomplish this goal we have used GWAS to map causal loci for variation in DNA methylation which we refer to as methylQTL. Understanding the types of genetic variants that can give rise to methylation variants is a major focus of our lab.<br /><strong><br />2. EpiGWAS<br /></strong>EpiGWAS is a variation of traditional GWAS that uses methylation polymorphisms as markers to test for association with a particular phenotype as opposed to the traditional approach which utilizes DNA polymorphisms. We are applying EpiGWAS to populations of plants to determine how much phenotypic variation results from DNA methylation polymorphisms. <br /><strong><br />3. Comparative Epigenomics<br /></strong>We are using comparative epigenomic approaches to understand both the variable and also the invariable portions of epigenomes by profiling DNA methylomes, histone tail modifications and RNAs from a variety of flowering plant species. Understanding why certain genes and gene families are hypervariably targeted by DNA methylation is important for revealing common functions in key biological processes. </div></div></div></div>

		</div><!-- End content -->
		<div class="clearer"></div>
	</div><!-- End main content wrapper -->
	<div id="footer"><!-- Start Footer -->
		<p></p>
		<div id="breadcrumbcontainer"><!-- Start the breadcrumb wrapper -->
			
		</div><!-- End breadcrumb -->
	</div><!-- End Footer -->
</div><!-- End container -->
</body>
</html>
